lib/aws/lex/conversation/type/intent.rb

```diff
@@ -22,6 +22,7 @@ class Intent
             end
 
             instance.raw_slots.each_with_object(default_hash) do |(key, value), hash|
+              value ||= { shape: 'Scalar' }
```

Comment:
Do you think we could add this to the `default_hash` instance defined a few lines above instead?

Reply:
We spoke about this and my suggestion won't work here. Lex will provide a slot key, but the value for the slot will be `nil`.